### DISCUSSION TITLE
How we can migrate Loopback 3 application to Loopback 4 ? What if we stick to Loopback 3, is there any issue or drawback with that

### DISCUSSION THREAD

**Discussion by Astha-Anand** [loopbackio/loopback-next#8111] [LB3 Migration] [answered]
Currently, our application is built in Loopback 3. What if we stay with Loopback 3. Is there any issue with sticking to loopback 3. Or if loopback 3 migration to Loopback 4 is mandatory, then how we can migrate to **loopback 4**. As we are facing issues in migration to **Loopback 4**.

Current Issue :

Unhandled rejection Error: Model "modelName" uses unknown mixin: CascadeDelete at MixinProvider.applyMixin

**Comment by achrinza** [comment, marked as answer]
See: https://stackoverflow.com/a/63750788/5019371
